config/software/datadog-agent.rb

```diff
@@ -64,7 +81,21 @@
       copy 'packaging/supervisor_32.conf', '/etc/dd-agent/supervisor.conf'
     end
     copy 'datadog.conf.example', '/etc/dd-agent/datadog.conf.example'
+    mkdir "/etc/dd-agent/conf.d/auto_conf"
```

Comment:
`auto_conf` is used there, whereas a few lines after it's `autoconf`. Which one is the good one?